### PR TITLE
Fix a couple of XML issues

### LINKED ISSUE #1444: HwTauLink: Bad options list for ComSpeed
- Seen in version (release version or githash): since 3fad4c6eb6d1c8b6d56e44524bb5e62d4c520d92

Missing outer `<options>` element.
https://github.com/d-ronin/dRonin/blob/next/shared/uavobjectdefinition/hwtaulink.xml#L31-L41

### PR DESCRIPTION
**Pull request by mlyle**
Fixes #1444

### CODE REVIEW

File: shared/uavobjectdefinition/hwlux.xml [hunk 1]
```diff
@@ -81,8 +81,10 @@
 		<field name="USB_HIDPort" units="function" type="enum" elements="1" parent="HwShared.USB_HIDPort" defaultvalue="USBTelemetry">
 			<description/>
 		</field>
-		<field name="USB_VCPPort" units="function" type="enum" elements="1" parent="HwShared.USB_VCPPort" defaultvalue="Disabled"><description>Virtual COM Port protocol</description>ption/&gt;
-			<options><option>Disabled</option><option>USBTelemetry</option><option>ComBridge</option><option>DebugConsole</option></options></field>
+		<field name="USB_VCPPort" units="function" type="enum" elements="1" parent="HwShared.USB_VCPPort" defaultvalue="Disabled">
+			<description>Virtual COM Port protocol</description>
+			<options><option>Disabled</option><option>USBTelemetry</option><option>ComBridge</option><option>DebugConsole</option></options>
```
Comment (tracernz) [inline shared/uavobjectdefinition/hwlux.xml:86]: Hmmm, not a fan of this, makes the diff more difficult when you change things.

Comment (mlyle) [reply shared/uavobjectdefinition/hwlux.xml:86]: i was just fixing the correctness issue...

Comment (tracernz) [reply shared/uavobjectdefinition/hwlux.xml:86]: Yep, no prob. :)

**Comment by tracernz** [comment]
LGTM

**Pull request merged**